src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -120,10 +120,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		DeadmanNotificationManager(coronaTestService: coronaTestService).scheduleDeadmanNotificationIfNeeded()
 
 		consumer.didFailCalculateRisk = { [weak self] error in
-			guard self?.store.isOnboarded == true else {
-				return
+			if self?.store.isOnboarded == true {
+				self?.showError(error)
 			}
-			self?.showError(error)
```

Comment:
Just wondering what is the difference now? The logic flow is the same isn't it? 🤔

Reply:
No, because in the guard we just returned without nothing, but in the normal flow, we expect `handleQuickActions(with: launchOptions)`

Reply:
handleQuickActions will not be reached either way. Because the return is in a closure.
But I think the if is better readable. So let's keep it.

Reply:
you are completely correct.